src/arm/cpu.rs

```diff
@@ -0,0 +1,28 @@
+use cfg_if::cfg_if;
+use libc;
+
+pub const DAV1D_ARM_CPU_FLAG_NEON: libc::c_uint = 1 << 0;
+pub const NEON_HWCAP: libc::c_ulong = 1 << 12;
+
+#[no_mangle]
+#[cold]
+pub unsafe extern "C" fn dav1d_get_cpu_flags_arm() -> libc::c_uint {
+    let mut flags = 0;
+
+    // TODO: Support Android by parsing `/proc/cpuinfo` the way the original C does.
```

Comment:
Couldn't you insert the android case and have it panic? That way, we're sure this cannot silently fail on Android.

Reply:
Good call, done.